application/common/models/User.php

```diff
@@ -168,7 +168,18 @@ public function updateProfileIfNeeded($firstName, $lastName, $username, $email)
      */
     public function getMaskedMethods()
     {
-        $methods = [];
+        /*
+         * Include primary email address
```

Comment:
I think best to just leave them out. We return boolean hasSupervisor and hasSpouse so the frontend can just tell us "supervisor" or "spouse", but I'm think it best not to display masked versions of the values. I might be convinced otherwise given a good use case, but for now I'll keep it limited.